pkgs/development/python-modules/pyxbe/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "pyxbe";
+  # https://github.com/mborgerson/pyxbe/issues/10
+  version = "20210208-unstable";
```

Comment:
There are releases https://github.com/mborgerson/pyxbe/releases.

Reply:
Yes, there are release. The last release was done in January 2020 and the the repo contains 8 commits which add new features.

Set it to the current release as the `cle` tests are passing.